examples/scripts/ConfigureRemotingForAnsible.ps1

```diff
@@ -258,6 +259,15 @@ Else
     Write-Verbose "Basic auth is already enabled."
 }
 
+# Check for CredSSP authentication
```

Comment:
This is great, but would prefer if it were a switch- some places are actively against CredSSP, so we probably don't want to enable by default (though one could make the same argument about Basic I suppose).